pytest_minio_mock/plugin.py

```diff
@@ -46,24 +49,32 @@ class MockMinioObject:
     Attributes:
         object_name (str): The name of the object.
         data (bytes or io.BytesIO): The data the object contains.
+        version_id (str, optional): The version of the object.
     """
 
-    def __init__(self, object_name, data):
+    def __init__(self, object_name, data, version_id=None):
         """
         Initialize the MockMinioObject with a name and data.
 
         Args:
             object_name (str): The name of the object.
             data (bytes or io.BytesIO): The data the object contains.
+            version_id (str, optional): The version of the object.
```

Comment:
Why allowing version_id=None?
Why not always have a version_id?
Even if versionning is disabled, giving the object a version_id should not be a problem.
that would simplify `get_object()` and `put_object()` the case when buckets already had objects before versionning is ENABLED (by accessing [bucket_name][object_name] last version `...[object_name].keys())[-1]` if versionning is not ENABLED or if version_id was not provided, and [bucket_name][object_name][version_id] if versionning is enabled)

What do you think?

Reply:
Indeed, it is much more simple. I'll do that.